compiler/src/dotty/tools/dotc/core/Flags.scala

```diff
@@ -563,6 +563,9 @@ object Flags {
   /** An inline parameter */
   final val InlineParam = allOf(Inline, Param)
 
+  /** An enum case */
+  final val EnumCase = allOf(Enum, Case)
```

Comment:
The documentation for `Enum` needs to be updated:

```Scala
   /** Symbol is a Java enum */
   final val Enum = commonFlag(40, "<enum>")
```

Reply:
That's done in the latest commit.